_posts/2021-07-01-quarkus-2x-platform-quarkiverse-registry.adoc

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: 'The New Quarkus 2.x Platform Model and Ecosystem Support in the Dev Tools'
+date: 2021-07-01
+tags: release
+synopsis: 'The Quarkus 2.0.0.Final release introduces the new Quarkus platform model and the dev tools that make it easy to discover and work with the Quarkus extension ecosystem.'
+author: aloubyansky
+---
+
```

Comment:
```suggestion
= Platforms and Streams: a new way to discover Quarkus extensions

As you may have seen from recent Quarkus Insights episodes (#48, #55 and #57, specifically), we've made a number of changes to our tools in the 2.x stream to improve the developer experience.  When 2.0 was released, a few sharp-eyed Quarkus client users spotted some new options, and we're excited to explain what those are all about with the release of Quarkus 2.1.

**TL;DR:** We have a whole new set of more granular BOMs that you can use instead of the big `quarkus-universe-bom`. We have deployed a registry service that aligns and manages platform compatibility across these more granular BOMs, and our tools (maven, gradle, and client) know how to work with this registry to simplify the discovery of extensions that work with your project.

As of the 2.1.0.Final release, the Quarkus client uses this registry by default to resolve the Quarkus platform used by your project.

[source, bash]
----
# The client will create a project with the latest/recommended quarkus release
$ quarkus create
# You can use the --stream parameter to narrow to the latest release 
$ quarkus create -S 2.0
$ quarkus create -S 2.1
----

```